Official build break: CoreCLR asset path missing '/' in installer subset

Example:

https://dev.azure.com/dnceng/internal/_build/results?buildId=493349&view=logs&j=03f0c49c-0e1a-5166-19f6-fc865fe47495&t=8ffb5fe8-216f-590f-db83-b0e95984e140&l=1033

> `F:\workspace\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\5.0.0-beta.20071.3\tools\Sign.proj(81,5): error MSB4018: System.IO.FileNotFoundException: Could not find file 'F:\workspace\_work\1\s\artifacts\transport\coreclrSystem.Private.CoreLib.dll'.`

Note `transport\coreclrSystem.Private.CoreLib.dll`

I think this is because of missing normalization here (https://github.com/dotnet/runtime/pull/1934):

https://github.com/dotnet/runtime/blob/fd7a5e8cf1748b7aefa90dd8e6966af2cbb4fe8b/eng/liveBuilds.targets#L21-L23

Because that variable is used in signing here:

https://github.com/dotnet/runtime/blob/fd7a5e8cf1748b7aefa90dd8e6966af2cbb4fe8b/eng/Signing.props#L39

I think normalizing would fix the break.

@dotnet/runtime-infrastructure

--- a/eng/liveBuilds.targets
+++ b/eng/liveBuilds.targets
@@ -19,8 +19,8 @@
     <LibrariesAllConfigurationsArtifactsPath Condition="'$(LibrariesAllConfigurationsOverridePath)' != ''">$([MSBuild]::NormalizeDirectory('$(LibrariesAllConfigurationsOverridePath)'))</LibrariesAllConfigurationsArtifactsPath>
 
     <!-- Honor the RuntimeArtifactsPath property. -->
-    <CoreCLRArtifactsPath Condition="'$(CoreCLRArtifactsPath)' == ''">$(RuntimeArtifactsPath)</CoreCLRArtifactsPath>
-    <MonoArtifactsPath Condition="'$(MonoArtifactsPath)' == ''">$(RuntimeArtifactsPath)</MonoArtifactsPath>
+    <CoreCLRArtifactsPath Condition="'$(CoreCLRArtifactsPath)' == ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</CoreCLRArtifactsPath>
+    <MonoArtifactsPath Condition="'$(MonoArtifactsPath)' == ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</MonoArtifactsPath>
   </PropertyGroup>
 
   <!--

--- a/eng/liveBuilds.targets
+++ b/eng/liveBuilds.targets
@@ -19,8 +19,8 @@
     <LibrariesAllConfigurationsArtifactsPath Condition="'$(LibrariesAllConfigurationsOverridePath)' != ''">$([MSBuild]::NormalizeDirectory('$(LibrariesAllConfigurationsOverridePath)'))</LibrariesAllConfigurationsArtifactsPath>
 
     <!-- Honor the RuntimeArtifactsPath property. -->
-    <CoreCLRArtifactsPath Condition="'$(CoreCLRArtifactsPath)' == ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</CoreCLRArtifactsPath>
-    <MonoArtifactsPath Condition="'$(MonoArtifactsPath)' == ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</MonoArtifactsPath>
+    <CoreCLRArtifactsPath Condition="'$(CoreCLRArtifactsPath)' == '' and '$(RuntimeArtifactsPath)' != ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</CoreCLRArtifactsPath>
+    <MonoArtifactsPath Condition="'$(MonoArtifactsPath)' == '' and '$(RuntimeArtifactsPath)' != ''">$([MSBuild]::NormalizeDirectory('$(RuntimeArtifactsPath)'))</MonoArtifactsPath>
   </PropertyGroup>
 
   <!--

